rpm/packagescanner.go

```diff
@@ -129,7 +132,7 @@ func (ps *Scanner) ScanContext(ctx context.Context, layer *claircore.Layer) ([]*
 	found := make([]string, 0)
 	for k, score := range possible {
 		if score == len(dbnames) {
-			found = append(found, k)
+			found = append(found, filepath.Join("/", k))
```

Comment:
Isn't this the case for **just** RHEL containers? Not other RPM based distros?

Reply:
No, this makes sure the paths are absolute.